_posts/en/posts/2016-06-24-segwit-next-steps.md

```diff
@@ -0,0 +1,166 @@
+
+**Lock-in:** once 95% of blocks in a 2,016-block long retarget period have signaled that their miners are ready to enforce segwit, segwit will lock-in -- meaning that unless the blockchain is rolled back at that point, segwit will become active (see next point).
+
+**Activation:** 2,016 blocks (about two weeks) after segwit is locked-in, it will activate.  That means all full nodes running segwit-aware code will begin requiring miners to enforce the new segwit consensus rules.
+
+**Wallets upgrade:** similar to the P2SH soft fork in 2012, after segwit activates it will not immediately be safe for wallets to upgrade to support segwit.  That’s because spends from segwit transactions look like unsecured transactions to older nodes, so if the blockchain is forked soon after segwit activates, those spends could be placed in an earlier block that is not subject to segwit's rules.  For this reason, it is suggested that wallets avoid upgrading for a few weeks after segwit activates.  Allowing that extra time to pass provides extra security to wallet users, although anyone who wants to test with a small amount of money they can afford to lose can begin spending as soon as segwit activates. Users can also begin testing immediately using testnet or regtest with the proposed segwit code or (when available) any release containing segwit.
+
+## How segwit will affect you
+
+- **Miners** who choose to run segwit will have the responsibility of ensuring that they're ready to enforce it by upgrading their full validation nodes to segwit-enforcing code.  When they’ve done this and performed whatever testing they believe is prudent, they can begin signaling support for segwit.
+
+- **Full node operators** can continue using their existing nodes but they are recommended to upgrade to a segwit-enforcing version.  If any miners end up producing blocks that are invalid according to the segwit rules after segwit activates, upgraded full nodes will automatically reject those blocks, ensuring that those upgraded full node users see accurate confirmation counts.
+
+    This upgrade is especially important for anyone, such as a business, who accepts transactions with low numbers of confirmations.
+
+- **Bitcoin Core wallet users** can continue using their existing nodes.  Even if you upgrade, you will see no changes beyond those described above.  The code expected to be released in 0.12.2 does not begin generating segwit receiving addresses by default.
+
+- **Users of other wallets** can continue using their existing wallets.  It is recommended that lightweight wallet users always wait for several confirmations when receiving significant amounts of money, so no extra waiting is expected to be required here.
+
+    When you have the opportunity to upgrade to a version of your wallet that supports segwit, you may find that the transaction fee you have to pay is slightly lower when you spend bitcoins you received after upgrading to segwit.
```

Comment:
maybe add a bit of explanation here: the transaction size counting towards the fee is smaller because the witness is external